src/rdkafka_conf.c

```diff
@@ -1013,6 +1013,38 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
           "the client's OAUTHBEARER token.",
           _UNSUPPORTED_OAUTHBEARER
         },
+        { _RK_GLOBAL, "sasl.oauthbearer.method", _RK_C_STR,
+          _RK(sasl.oauthbearer.method),
+          "Enable the functionality of the OAuth/OIDC login callback handler.",
+          _UNSUPPORTED_OAUTHBEARER
```

Comment:
Since OAUTHBEARER OIDC also depends on WITH_CURL I think you need to add a new `_UNSUPPORTED_OIDC` to be used here instead, with the appropriate .unsupported message if not available.